index.html

```diff
@@ -171,15 +170,59 @@ <h2>Multi-Post Stories </h2>
                 <button class="button" type="button" onclick="">See Project</button>
             </div>
         </div>
-
-
-
-
-
-
-
-
-
+<footer>
+    <div class="template-footer">
+        <h1>CONTACT US</h1>
+        <p>If you have an application you are <br>
+            interested in developing, a feature that you <br> 
+            need built or a project that needs coding. <br>
+            I’d love to help with it </p>
+
+            <div class="form-section">
+                <form class="form-submit" action="https://formspree.io/f/form_id" method="post">
```

Comment:
- [ ] Please create a form and integrate it with [formspree service](https://formspree.io/html) so the user can send you a message and possibly invite you to a job. Your form is not working with this action link.